stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -68,11 +70,16 @@ public static void main(String[] args) {
 
   @Override
   public void prepare() throws Exception {
+    if (mParameters.mNumFiles % mParameters.mNumDirs != 0) {
+      throw new IllegalArgumentException(String
+          .format("Number of files (%d) must be a multiple of the number of directories (%d)",
+              mParameters.mNumFiles, mParameters.mNumDirs));
+    }
     for (Integer numOfThreads: mParameters.mThreads) {
-      if (numOfThreads > mParameters.mNumFiles) {
+      if (mParameters.mNumFiles % numOfThreads != 0) {
```

Comment:
In our case, better to not allowing thread array list. 
I took a look at Fuse logics, there are many operating system cache involved in reading, which means that when the application read a file for the first time, it goes to Fuse, but then the file may be cached in the operating system and no actual read happens at AlluxioFuse. 
We should just change mThreads to a single integer